docs/src/pages/components/progress/CircularWithValueLabel.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { CircularProgress, Typography, Box } from '@material-ui/core';
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: 'flex',
+    '& > * + *': {
```

Comment:
You are right, thanks for the hint. I just copy it from the other examples :)
I will also change it in the other examples so that it is consistent...

Reply:
> I will also change it in the other examples so that it is consistent...

That would be awesome!

Reply:
Jmm... I just saw that in every component example (e.g. [Buttons](http://localhost:3000/components/progress)), the margins are done with  `'& > * + *'`. 
I think this is done so that the snippet shown at the beginning is as simple as possible.
So I think it is better to leave it like that so that is consistent with all other components...

Reply:
Sure, let's follow up later.